.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
This file shouldn't be version controlled

Reply:
Without these I do not have access to the smartcar library, its a weird mechanism that VS Code uses

Reply:
Sure, you can keep them but you shouldn't version control them. They even contain paths to **your** machine.

Reply:
Understood, will remove them in future pull requests! Thanks for the heads up